app/src/main/java/com/hedvig/app/feature/onbarding/ChoosePlanRepository.kt

```diff
@@ -0,0 +1,18 @@
+package com.hedvig.app.feature.onbarding
+
+import android.content.Context
+import com.apollographql.apollo.coroutines.await
+import com.hedvig.android.owldroid.graphql.ChoosePlanQuery
+import com.hedvig.app.ApolloClientWrapper
+import com.hedvig.app.util.apollo.defaultLocale
+import com.hedvig.app.util.apollo.toLocaleString
+
+class ChoosePlanRepository(
+    private val apolloClientWrapper: ApolloClientWrapper,
+    context: Context
+) {
+    private val locale = defaultLocale(context).toLocaleString()
+
+    suspend fun getBundles() =
```

Comment:
```suggestion
    suspend fun bundles() =
```

This is more in line with how we usually name these functions.